server/server.js

```diff
@@ -75,6 +75,8 @@ app.use((err, req, res) => {
 
 app.use(errorHandler);
 
-app.listen(port, () => console.log(`Server running on port ${port}`));
+if (process.env.TEST_ENV !== "true") {
```

Comment:
Added a minor conditional to be able to instantiate the server, i.e. set them up and tear them down. Development environments and production environments will have a falsy conditional; thus, be unaffected.

this was a necessary change to be able run multiple test suites with a real server.